### PR TITLE
a bit of refactoring

### PR DESCRIPTION
**Pull request by Vane11ope**

### CODE REVIEW

File: libr/core/panels.c [hunk 1]
```diff
@@ -106,6 +106,28 @@ static const char **menus_sub[] = {
 	NULL
 };
 
+static void Layout_run(RPanels *panels);
+static void Panel_print(RConsCanvas *can, RPanel *panel, int color);
```
Comment (xarkes) [inline libr/core/panels.c]: Why uppercase letter?

Comment (Vane11ope) [reply libr/core/panels.c]: it's how it had been before me

Comment (Vane11ope) [reply libr/core/panels.c]: but in fact, those uppercase look weird to me too so i fixed 'em now, thanks :)

**Comment by xarkes** [comment]
I'm thinking and maybe it would be better to apply the same style everywhere.
So `static` functions in camelCase so every panel_something should be renamed to panelSomething and have the same format.
Right now there is `static void panelPrint` and `static void panel_prompt` which does not make much sense.
The only exception is `r_core_panels_refresh` I would say.

**Comment by Vane11ope** [comment]
fair enough i'd fix like that

File: libr/core/panels.c [hunk 2]
```diff
@@ -106,7 +106,29 @@ static const char **menus_sub[] = {
...
-static void Panel_print(RConsCanvas *can, RPanel *panel, int color) {
+static void layoutRun(RPanels *panels);
+static void panelPrint(RConsCanvas *can, RPanel *panel, int color);
+static void addPanelFrame(RPanels* panels, const char *title, const char *cmd);
+static void check_stackbase(RCore *core);
+static void cursor_left(RCore *core);
+static void cursor_right(RCore *core);
+static void delcurpanel(RPanels *panels);
+static void doRefresh(RPanels *panels);
+static bool handle_cursormode(RCore *core, const int key);
```
Comment (radare) [inline libr/core/panels.c]: all static functions should be camelcase imho

Comment (Vane11ope) [reply libr/core/panels.c]: agreed and fixed like that

**Comment by Vane11ope** [comment]
done it

**Pull request merged**